cli/internal/cloudcmd/create.go

```diff
@@ -23,15 +23,14 @@ import (
 	"github.com/edgelesssys/constellation/v2/cli/internal/terraform"
 	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 	"github.com/edgelesssys/constellation/v2/internal/config"
-	"github.com/edgelesssys/constellation/v2/internal/constants"
 	"github.com/edgelesssys/constellation/v2/internal/imagefetcher"
 )
 
 // Creator creates cloud resources.
 type Creator struct {
 	out                io.Writer
 	image              imageFetcher
-	newTerraformClient func(ctx context.Context) (tfResourceClient, error)
+	newTerraformClient func(ctx context.Context, workspace string) (tfResourceClient, error)
```

Comment:
why are you using the big client again? I think we should try to get rid of it. My idea was to actually get rid of this bigger client and separate them (tfResource / tfIAM). A cluster `create` should not be able to access IAM functionality.

Reply:
I'm not sure what you are talking about?
I simply added a string as argument to the function. The goal of this PR is not to refactor any client usage. Those were left untouched.